index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inter:400,600,500,800" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Welcome Header -->
+   <header class="header">
+       <h1>Welcome</h1>
+       <i class="fa fa-bars" aria-hidden="true"></i>
```

Comment:
- [ ] [REQUIRED] I think this hamburger button should be enclosed in a nav tag because the nav tag is reserved for primary navigation areas, like the main menu across the top of the page or section.😊